compose.yaml

```
@@ -21,9 +21,9 @@ services:
 
   database_default:
     # Select one of the following db configurations for the database
-    image: postgres:13.5-alpine
+    image: postgres:15-alpine
     ports:
-      - "5432:5432/tcp"  # allow your local dev env to connect to the db
+      - "${EXT_PG_PORT-}:5432/tcp"  # allow your local dev env to connect to the db if variable set
```

```suggestion
      - "${DB_PORT-}:5432/tcp"  # allow your local dev env to connect to the db if variable set
```
I prefer a more generic, shorter db port name.